src/main/java/io/fabric8/maven/docker/wait/WaitUtil.java

```diff
@@ -1,5 +1,11 @@
 package io.fabric8.maven.docker.wait;
 
+import io.fabric8.maven.docker.access.DockerAccess;
```

Comment:
Actually I'd prefer to add a `IsRunningChecker` which is put first in the checker chain. This checker can have access to the `dockerAccess` (ike the `LogWaitChecker` has). Then, if the container is not running I would throw a `WaitResultException`, generalising on the `WaitTimeoutException`

```java
publi class WaitResultException extend Exception {

    private WaitResultException.Reason reason;

   enum Reason { timeout, noRunning }
   ....
}
```